docs/csharp/tutorials/intro-to-csharp/hello-world.yml

```diff
@@ -221,8 +221,8 @@ items:
     > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
     
 - content: |
-    You've completed the "Hello C#" introduction to C# tutorial.
+    You've completed the "Hello C#" introduction to C# tutorial. You can click the **Next** link below to start the **Numbers in C# interactive tutorial**, or you can visit the [.NET site](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial) to download the .NET Core SDK, create a project on your machine, and keep coding. The "Keep Learning" step bring you back to these tutorials.
```

Comment:
```suggestion
    You've completed the "Hello C#" introduction to C# tutorial. You can click the **Next** link below to start the **Numbers in C# interactive tutorial**, or you can visit the [.NET site](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial) to download the .NET Core SDK, create a project on your machine, and keep coding. The "Keep Learning" step will bring you back to these tutorials.
```

Reply:
I think you are missing a `will` from this text

Reply:
Good catch. I used present tense instead of your suggestion "brings" vs. "will bring"